app/services/support_interface/application_choices_export.rb

```diff
@@ -68,5 +69,24 @@ def relevant_applications
         .where.not(submitted_at: nil)
         .order('submitted_at asc')
     end
+
+    def format_structured_rejection_reasons(structured_rejection_reasons)
+      return nil if structured_rejection_reasons.blank?
+
+      select_high_level_rejection_reasons(structured_rejection_reasons)
+      .keys
+      .map { |reason| format_reason(reason) }
+      .join("\n")
+    end
+
+    def select_high_level_rejection_reasons(structured_rejection_reasons)
+      structured_rejection_reasons.select { |reason, value| value == 'Yes' && reason.include?('_y_n') }
+    end
+
+    def format_reason(reason)
+      reason
+      .delete_suffix('_y_n')
+      .humanize
```

Comment:
There are labels for these in `reasons_for_rejection.yml` but not very logically organised (at least not to me). So again this is the best way to do this for now but something we might want to revisit.